CMakeLists.txt

```diff
@@ -235,6 +258,56 @@ if(COVERALLS)
     "${PROJECT_SOURCE_DIR}/cmake")
 endif()
 
+#===============================================================================
+# Automatic code formatting using clang-format.
+#
+find_program(
+  CLANG_FORMAT_EXECUTABLE
+  NAMES clang-format clang-format-3.9 clang-format-3.8
+)
+
+if (CLANG_FORMAT_EXECUTABLE)
+
+  get_property(formatting_files GLOBAL PROPERTY CODE_FORMATTING_SOURCES)
+  list(LENGTH formatting_files formatting_files_length)
+
+  if (formatting_files)
+
+    add_custom_target(format
+        COMMAND ${CMAKE_COMMAND} -E echo "Formatting code style of"
+            "${formatting_files_length} files... "
+        COMMAND ${CLANG_FORMAT_EXECUTABLE} -style=file -i ${formatting_files}
+        COMMAND ${CMAKE_COMMAND} -E echo "Done."
+        DEPENDS ${CLANG_FORMAT_EXECUTABLE}
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    )
+    add_custom_target(check-format
+        COMMAND ${CMAKE_COMMAND} -E echo "Checking code style of
+            ${formatting_files_length} files... "
+        COMMAND ${CMAKE_SOURCE_DIR}/tools/check_code_format.sh
+            ${CLANG_FORMAT_EXECUTABLE} ${formatting_files}
+        COMMAND ${CMAKE_COMMAND} -E echo "Done."
+        DEPENDS ${CLANG_FORMAT_EXECUTABLE}
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    )
+
+  else()
+
+    add_custom_target(code-formatting
```

Comment:
Why is this target named `code-formatting` but the target in the other branch of this `if-else` block called `format`?